app/views/WishList/index.tsx

```diff
@@ -0,0 +1,203 @@
+                        className={styles.details}
+                        heading={title}
+                    >
+                        <div className={styles.headerDescription}>
+                            <TextOutput
+                                label="Author"
+                                value={authorsDisplay}
+                            />
+                            <TextOutput
+                                label="Price (NPR)"
+                                valueType="number"
+                                value={price}
+                            />
+                            <div className={styles.quantity}>
+                                <TextOutput
+                                    label="Quantity"
+                                    valueType="number"
+                                />
+                                <NumberInput
+                                    name="quantity"
+                                    value={undefined}
+                                    onChange={undefined}
+                                />
+                            </div>
+                        </div>
+                    </Container>
+                    <div className={styles.wishListButton}>
+                        <Button
+                            name={undefined}
+                            onClick={undefined}
+                            variant="secondary"
+                            icons={<FaShoppingCart />}
+                            autoFocus
+                        >
+                            Add to cart
+                        </Button>
+                        <Button
+                            name={undefined}
+                            onClick={() => removeBook(id)}
+                            variant="secondary"
+                            icons={<AiTwotoneDelete />}
+                            autoFocus
+                        >
+                            Remove
+                        </Button>
+                    </div>
+                </div>
+            </div>
+        </>
+    );
+}
+
+function WishList() {
+    const [page, setPage] = useState<number>(1);
+    const [pageSize, setPageSize] = useState<number>(10);
+    const [deleteWishlist] = useMutation(REMOVE_WISH_LIST);
+
+    const { data, refetch, loading } = useQuery<
+        WishListQuery,
+        WishListQueryVariables
+    >(WISH_LIST, {
+        variables: { page, pageSize },
+    });
+
+    const deleteBook = (_id: number) => {
+        deleteWishlist({ variables: { id: _id } });
+        refetch();
+    };
+
+    useEffect(() => {
+        if (!loading && data?.wishList) {
+            if (data.wishList.page != null && data.wishList.page) {
+                setPage(data.wishList.page);
+            }
+            if (data.wishList.pageSize != null && data.wishList.pageSize) {
+                setPageSize(data.wishList.pageSize);
+            }
+        }
+    }, [loading]);
```

Comment:
Can you please describe what this block does?

Reply:
For pagination to be implemented in the future.